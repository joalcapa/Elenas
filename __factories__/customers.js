export const customers = [
    {
        id: 1,
        firstName: 'Test first name',
        lastName: 'Test last name',
        email: 'test@test.com',
        cedula: '1000000',
        address: 'Test address',
        cellphone: '+00587458',
    },
    {
        id: 2,
        firstName: 'Test first name 2',
        lastName: 'Test last name 2',
        email: 'test2@test.com',
        cedula: '200000',
        address: 'Test address 2',
        cellphone: '+002556421',
    },
];