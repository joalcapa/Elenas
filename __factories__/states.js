export const states = [
    {
        id: 1,
        name: 'Test state',
        displayName: 'Test state',
        shortCode: 'TestCode',
        shippingZone: {
            name: 'Test',
            cost: 'test',
        },
        cities: [
            {
                id: 1,
                name: 'Test city',
                shortCode: 'TestCode',
                displayName: 'Test',
                codAvailable: 'Test',
                isCapital: false,
            } ,
        ],
    },
    {
        id: 2,
        name: 'Test state 2',
        displayName: 'Test state 2',
        shortCode: 'TestCode2',
        shippingZone: {
            name: 'Test2',
            cost: 'test2',
        },
        cities: [
            {
                id: 3,
                name: 'Test city 3',
                shortCode: 'TestCode3',
                displayName: 'Test',
                codAvailable: 'Test',
                isCapital: false,
            },
        ],
    },
];