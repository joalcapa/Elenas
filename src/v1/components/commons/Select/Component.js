import React from 'react';
import PropTypes from 'prop-types';
import {TouchableOpacity, Modal, View, ScrollView} from 'react-native';

import TextField from '../../../components/commons/TextField';
import Button from '../../../components/commons/Button';

import styles from './styles';

/**
 * Componente que renderiza los items de un select basico.
 */
const Component = (props) => (
    <>
        <Modal
            animationType="none"
            transparent={true}
            visible={props.isVisibleSelect}
            onRequestClose={props.hiddenSelect}>
            <View style={styles.container}>
                <View style={styles.content}>
                    <ScrollView>
                        {props.options.map((option) => (
                            <TouchableOpacity
                                key={`Select${option.label}`}
                                style={styles.item_content}
                                onPress={() => {
                                    props.onSelect(option);
                                    props.hiddenSelect();
                                }}>
                                <TextField text={option.label} />
                            </TouchableOpacity>
                        ))}
                    </ScrollView>
                </View>
                <Button
                    isGreenButton
                    text={'Salir'}
                    onPress={props.hiddenSelect}
                />
            </View>
        </Modal>
        <TouchableOpacity onPress={props.options.length > 0 ? props.showSelect : null} style={styles.button_content}>
            <TextField text={props.value ? props.value : 'Seleccionar'} />
        </TouchableOpacity>
    </>
);

Component.propTypes = {
    /**
     * Valor del select.
     */
    value: PropTypes.string,
    /**
     * Opciones del select.
     */
    options: PropTypes.array,
    /**
     * Funcion que devuelve el dato seleccionado.
     */
    onSelect: PropTypes.func,
    /**
     * Funcion que muestra el selector (Modal).
     */
    showSelect: PropTypes.func,
    /**
     * Funcion que oculta el selector (Modal).
     */
    hiddenSelect: PropTypes.func,
    /**
     * Determina si el selector se encuentra visible.
     */
    isVisibleSelect: PropTypes.bool,
};

Component.defaultProps = {
    value: '',
    options: [],
    onSelect: () => {},
    showSelect: () => {},
    hiddenSelect: () => {},
    isVisibleSelect: false,
};

export default Component;