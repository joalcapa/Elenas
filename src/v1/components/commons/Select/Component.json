{
  "description": "",
  "displayName": "Component",
  "methods": [],
  "props": {
    "value": {
      "type": {
        "name": "string"
      },
      "required": false,
      "description": "Valor del select.",
      "defaultValue": {
        "value": "''",
        "computed": false
      }
    },
    "options": {
      "type": {
        "name": "array"
      },
      "required": false,
      "description": "Opciones del select.",
      "defaultValue": {
        "value": "[]",
        "computed": false
      }
    },
    "onSelect": {
      "type": {
        "name": "func"
      },
      "required": false,
      "description": "Funcion que devuelve el dato seleccionado.",
      "defaultValue": {
        "value": "() => {}",
        "computed": false
      }
    },
    "showSelect": {
      "type": {
        "name": "func"
      },
      "required": false,
      "description": "Funcion que muestra el selector (Modal).",
      "defaultValue": {
        "value": "() => {}",
        "computed": false
      }
    },
    "hiddenSelect": {
      "type": {
        "name": "func"
      },
      "required": false,
      "description": "Funcion que oculta el selector (Modal).",
      "defaultValue": {
        "value": "() => {}",
        "computed": false
      }
    },
    "isVisibleSelect": {
      "type": {
        "name": "bool"
      },
      "required": false,
      "description": "Determina si el selector se encuentra visible.",
      "defaultValue": {
        "value": "false",
        "computed": false
      }
    }
  }
}
