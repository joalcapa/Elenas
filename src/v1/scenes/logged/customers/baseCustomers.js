import React, { Children } from 'react';
import PropTypes from 'prop-types';
import {View, StyleSheet} from 'react-native';

import Button from '../../../components/commons/Button';
import TextField, {BIG_TYPE} from '../../../components/commons/TextField';

const BaseCustomer = (props) => {
    const {title, Child, stackProps, options} = props;
    return (
        <View style={styles.container}>
            <View style={styles.nav_buttons}>
                {title !== '' && (
                    <TextField 
                        isTitle 
                        text={title} 
                        type={BIG_TYPE}
                    />)}
                <View style={styles.buttons_container}>
                    {options.isCreateButton && (
                        <View style={styles.button_content}>
                            <Button text='CREAR CLIENTE'/>
                        </View>)}
                    {options.isListButton && (
                        <View style={styles.button_content}>
                            <Button text='LISTA DE CLIENTES'/>
                        </View>)}
                </View>
            </View>
            <View style={styles.container}>
                <Child {...stackProps} />
            </View>
        </View>
    );
};

BaseCustomer.propTypes = {
    title: PropTypes.string,
    Child: PropTypes.any,
    options: PropTypes.shape({
        isCreateButton: PropTypes.bool,
        isListButton: PropTypes.bool,
    }),
    stackProps: PropTypes.shape({}),
};

BaseCustomer.defaultProps = {
    title: '',
    Child: () => <></>,
    options: {
        isCreateButton: false,
        isListButton: false,
    },
    stackProps: {},
};

const styles = StyleSheet.create({
    container: {
        paddingLeft: '20px',
        paddingTop: '10px',
    },
    buttons_container: {
        width: 400,
    },
    button_content: {
        display: 'flex',
        alignItems: 'flex-end',
        paddingRight: '10px',
    },
    nav_buttons: {
        display: 'flex',
        flexDirection: 'row',
        justifyContent: 'space-between',
    },
});

export default (Child, title = '', options = {}) => (stackProps) => (
    <BaseCustomer 
        title={title} 
        Child={Child} 
        options={options}
        stackProps={stackProps}
    />
);