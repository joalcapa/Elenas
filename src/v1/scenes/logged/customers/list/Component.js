import React from 'react';
import PropTypes from 'prop-types';
import {useHistory} from 'react-router-dom';
import {View} from 'react-native';

import TextField, {EXTRA_TYPE} from '../../../../components/commons/TextField';
import Button from '../../../../components/commons/Button';

import styles from './styles';

/**
 * Este componente permite listar la informacion de los usuarios.
 */
const ItemList = (props) => (
    <View style={styles.contain}>
        <View style={styles.BoxOne}>
            <TextField text={`${props.id}`} style={styles.textBox} type={EXTRA_TYPE}/>
        </View>
        <View style={styles.BoxOne}>
            <TextField text={props.firstName} style={styles.textBox} type={EXTRA_TYPE} />
        </View>
        <View style={styles.BoxOne}>
            <TextField text={props.lastName} style={styles.textBox} type={EXTRA_TYPE}/>
        </View>
        <View style={styles.BoxOne}>
            <TextField text={props.cedula} style={styles.textBox} type={EXTRA_TYPE}/>
        </View>
        <View style={styles.BoxOne}>
            <TextField text={props.email} style={styles.textBox} type={EXTRA_TYPE}/>
        </View>
        <View style={styles.BoxOne}>
            <TextField text={props.address} style={styles.textBox} type={EXTRA_TYPE}/>
        </View>
        <View style={styles.BoxOne}>
            <TextField text={props.cellphone} style={styles.textBox} type={EXTRA_TYPE}/>
        </View>
        <View style={styles.BoxOne}>
            {props.isEdit ?
                <Button text="Editar" onPress={props.onPress} /> :
                <TextField text="ACCION" style={styles.textBox} type={EXTRA_TYPE}/>
            }
        </View>
    </View>
);

ItemList.propTypes = {
    /**
     * Valor del id del cliente.
     */
    id: PropTypes.any,
    /**
     * Nombre del cliente.
     */
    firstName: PropTypes.string,
    /**
     * Apellido del cliente.
     */
    lastName: PropTypes.string,
    /**
     * Cedula del cliente.
     */
    cedula: PropTypes.string,
    /**
     * Correo del cliente .
     */
    email: PropTypes.string,
    /**
     * Direccion del cliente.
     */
    address: PropTypes.string,
    /**
     * Telefono del cliente.
     */
    cellphone: PropTypes.string,
    /**
     * Permite indicar si se editara el cliente.
     */
    isEdit: PropTypes.bool,
    /**
     * Funcion que edita el cliente.
     */
    onPress: PropTypes.func,
};

ItemList.defaultProps = {
    id: 'ID',
    firstName: 'NOMBRE',
    lastName: 'APELLIDO',
    cedula: 'CEDULA',
    email: 'EMAIL',
    address: 'DIRECCION',
    cellphone: 'TELEFONO',
    isEdit: false,
    onPress: () => {},
};

const Component = (props) => {
    let history = useHistory();

    const toCustomerEdit = (id) => {
        history.push(`/customer/${id}`);
    };

    return (
        <View style={styles.container}>
            <ItemList />
            {props.customers.map((customer) =>
                <ItemList
                    key={`Customer${customer.id}`}
                    isEdit 
                    {...customer} 
                    onPress={() => toCustomerEdit(customer.id)} 
                />)}
        </View>
    );
};

Component.propTypes = {
    /**
     * Array de clientes.
     */
    customers: PropTypes.array,
};

Component.defaultProps = {
    customers: [],
};

export default Component;