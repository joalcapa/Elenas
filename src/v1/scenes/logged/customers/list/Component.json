{
  "description": "",
  "displayName": "Component",
  "methods": [],
  "props": {
    "customers": {
      "type": {
        "name": "array"
      },
      "required": false,
      "description": "Array de clientes.",
      "defaultValue": {
        "value": "[]",
        "computed": false
      }
    }
  }
}
