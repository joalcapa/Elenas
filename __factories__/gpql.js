import {LOGIN} from '../src/kernel/v1/gql/mutations';
import {customers} from './customers';

export const responseGpql = [
    {
        request: {
            query: LOGIN,
            variables: {
                cellphone: customers[0].cellphone,
                password: 'password',
            },
        },
        result: {
            data: {
                login: {
                    token: 'auth token',
                    user: customers[0],
                },
            },
        },
    },
    {
        request: {
            query: LOGIN,
            variables: {
                cellphone: customers[0].cellphone,
                password: 'other password',
            },
        },
        result: {
            data: {
                login: {
                    __typename: 'ValidationErrors',
                },
            },
        },
    },
];