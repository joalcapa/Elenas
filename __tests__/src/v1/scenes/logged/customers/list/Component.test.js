import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import {create} from 'react-test-renderer';
import {configure} from 'enzyme';

import Component from '../../../../../../../src/v1/scenes/logged/customers/list/Component';
import {customers} from '../../../../../../../__factories__/customers';

configure({adapter: new Adapter()});
describe('Component List component', () => {
    test('render empty list component', () => {
        const tree = create(<Component />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('render customers list component', () => {
        const tree = create(<Component customers={customers} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});