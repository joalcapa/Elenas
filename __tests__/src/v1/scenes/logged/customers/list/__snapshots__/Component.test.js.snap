// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component List component render customers list component 1`] = `
<div
  className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-justifyContent-1777fci r-paddingBottom-1uwte3a r-paddingTop-1knelpx r-width-16k0tzm"
>
  <div
    className="css-view-1dbjc4n r-alignContent-uia4a0 r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        ID
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        NOMBRE
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        APELLIDO
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        CEDULA
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        EMAIL
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        DIRECCION
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        TELEFONO
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        ACCION
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-alignContent-uia4a0 r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        1
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        Test first name
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        Test last name
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        1000000
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        test@test.com
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        Test address
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        +00587458
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-l7f4p8 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-displayFlex-14n0d7j r-justifyContent-1777fci r-maxWidth-127gp16 r-paddingBottom-xd6kpl r-paddingTop-tskmnb r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr r-width-13qz1uu"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          className="css-text-901oao r-color-cqee49 r-fontSize-1b43r93"
          dir="auto"
          onClick={[Function]}
        >
          Editar
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-alignContent-uia4a0 r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        2
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        Test first name 2
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        Test last name 2
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        200000
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        test2@test.com
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        Test address 2
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        +002556421
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-l7f4p8 r-borderRadius-1xfd6ze r-cursor-1loqt21 r-displayFlex-14n0d7j r-justifyContent-1777fci r-maxWidth-127gp16 r-paddingBottom-xd6kpl r-paddingTop-tskmnb r-touchAction-1otgn73 r-transitionProperty-1i6wzkk r-userSelect-lrvibr r-width-13qz1uu"
        onClick={[Function]}
        onContextMenu={[Function]}
        onKeyDown={[Function]}
        style={
          Object {
            "WebkitTransitionDuration": "0s",
            "transitionDuration": "0s",
          }
        }
        tabIndex="0"
      >
        <div
          className="css-text-901oao r-color-cqee49 r-fontSize-1b43r93"
          dir="auto"
          onClick={[Function]}
        >
          Editar
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component List component render empty list component 1`] = `
<div
  className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-justifyContent-1777fci r-paddingBottom-1uwte3a r-paddingTop-1knelpx r-width-16k0tzm"
>
  <div
    className="css-view-1dbjc4n r-alignContent-uia4a0 r-alignItems-1awozwy r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        ID
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        NOMBRE
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        APELLIDO
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        CEDULA
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        EMAIL
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        DIRECCION
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        TELEFONO
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingBottom-1mi0q7o r-paddingLeft-1yt7n81 r-paddingRight-ry3cjt r-width-8hv5mx"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        onClick={[Function]}
        style={
          Object {
            "fontSize": "16px",
          }
        }
      >
        ACCION
      </div>
    </div>
  </div>
</div>
`;
